### PR TITLE
[spacefi-blp] add blp strategy

### PR DESCRIPTION
**Pull request by SpaceFinance**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @SpaceFinance Could you please add a readme to this strategy? will help users know more about it

**Comment by SpaceFinance** [comment]
[README.md](https://github.com/snapshot-labs/snapshot-strategies/files/14018698/README.md)

**Comment by SpaceFinance** [comment]
> Hi @SpaceFinance Could you please add a readme to this strategy? will help users know more about it

Added, please check

File: src/strategies/spacefi-blp/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'icepoint';
```
Comment (ChaituVR) [inline src/strategies/spacefi-blp/index.ts:5]:
```suggestion
export const author = 'SpaceFinance';
```

File: src/strategies/spacefi-blp/index.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/strategies/spacefi-blp/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
should add strategy to /src/strategies/index.ts like all other strategies

**Comment by SpaceFinance** [comment]
> should add strategy to /src/strategies/index.ts like all other strategies

Added Completed

**Comment by ChaituVR** [comment]
Tests are failing can take a look? try running `yarn test --strategy=spacefi-blp --more=500` in your local

**Comment by SpaceFinance** [comment]
> Tests are failing can take a look? try running `yarn test --strategy=spacefi-blp --more=500` in your local

Modified, please check

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**